pages/common/git-check-ignore.md

```diff
@@ -0,0 +1,20 @@
+# git check-ignore
+
+> Analyse and debug Git ignore / exclude (".gitignore") files.
+> More information: <https://git-scm.com/docs/git-check-ignore>.
+
+- Check whether one or more space-separated files or directories are ignored:
+
+`git check-ignore {{path/to/files_or_directories}}`
```

Comment:
Maybe it would be clearer to split this example into one that uses a single path and one that uses multiple paths?